(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'DAL' object has no attribute 'sala_matriculado'"
p3
sS'layer'
p4
S'/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7fbfafee7430, file "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 8>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/leonardo/Downloads/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 130, in <module>\n  File "/home/leonardo/Downloads/web2py/gluon/globals.py", line 419, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/leonardo/Downloads/web2py/gluon/tools.py", line 3982, in f\n    return action(*a, **b)\n  File "/home/leonardo/Downloads/web2py/applications/KidsAppManager/controllers/default.py", line 75, in entrada\n    query2=  db(db.cadastro.id == db.sala_matriculado.id).select(sala_matriculado.nome_sala)\n  File "/home/leonardo/Downloads/web2py/gluon/packages/dal/pydal/base.py", line 669, in __getattr__\n    return BasicStorage.__getattribute__(self, key)\nAttributeError: \'DAL\' object has no attribute \'sala_matriculado\'\n'
p11
s.